enterprise/static/enterprise/admin/manage_learners.js

```diff
@@ -18,5 +55,20 @@ $(
                 })
             }
         });
+        $("#id_course").on('input', function(evt){
+            /*
+            We have to do two-step input/blur because some browsers (looking at you, Safari)
+            don't properly send the .change() event when the input is filled with autocomplete.
+            */
+            evt.target.dirty = true;
+        });
+        $("#id_course").blur(function(evt){
```

Comment:
space around open bracket. many places above . 
nit: good to have : `evt` ~ `event`